chain/evm/compass.go

```diff
@@ -617,3 +840,111 @@ func (c compass) callCompass(
 	}
 	return c.evm.ExecuteSmartContract(ctx, c.chainID, *c.compassAbi, c.smartContractAddr, useMevRelay, method, arguments)
 }
+
+func (t compass) gravityRelayBatches(ctx context.Context, batches []chain.GravityBatchWithSignatures) error {
+	var gErr whoops.Group
+	logger := log.WithField("chainReferenceID", t.ChainReferenceID)
```

Comment:
In Pigeon, we can now use logging with proper call correlation. You can use it like this:

```suggestion
	logger := liblog.WithContext(ctx).WithField("chainReferenceID", t.ChainReferenceID)
```

Reply:
We do not have this in Paloma (yet), as both use different logging solutions.